pandas/tests/series/test_datetime_values.py

```diff
@@ -665,3 +667,23 @@ def test_setitem_with_different_tz(self):
             dtype=object,
         )
         tm.assert_series_equal(ser, expected)
+
+    @pytest.mark.parametrize(
+        "input_series, expected_output, expected_type",
+        [
+            [["2020-01-01"], [[2020, 1, 3]], "int32"],
+            [[pd.NaT], [[np.NaN, np.NaN, np.NaN]], "float64"],
+            [["2019-12-31", "2019-12-29"], [[2020, 1, 2], [2019, 52, 7]], "int32"],
+            [
+                ["2010-01-01", pd.NaT],
+                [[2009, 53, 5], [np.NaN, np.NaN, np.NaN]],
+                "float64",
+            ],
+        ],
+    )
+    def test_isocalendar(self, input_series, expected_output, expected_type):
+        ser = pd.Series(input_series, dtype="datetime64[D]")
```

Comment:
You can use `to_datetime` instead of `dtype="datetime64[D]"` here. (We don't actually support `"datetime64[D]"`)

Reply:
Thanks!